v2/examples/send/send_message.go

```diff
@@ -0,0 +1,26 @@
+package examples
+
+import (
+	"context"
+	"log"
+
+	"github.com/trycourier/courier-go/v2"
+)
+
+func sendMessage() {
+	client := courier.CreateClient("<YOUR_AUTH_TOKEN>", nil)
+
+	requestID, err := client.SendMessage(context.Background(), courier.SendMessageRequestBody{
+		Message: map[string]interface{}{
+			"template": "my-template",
```

Comment:
is this strictly typed or similar to how things are in Javascript?

Reply:
top level Message would be strict. but that bag would have anything.